recipes/libgeotiff/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class LibgeotiffConan(ConanFile):
+    name = "libgeotiff"
+    description = "Libgeotiff is an open source library normally hosted on top " \
+                  "of libtiff for reading, and writing GeoTIFF information tags."
+    license = ["MIT", "BSD-3-Clause"]
+    topics = ("conan", "libgeotiff", "geotiff", "tiff")
+    homepage = "https://github.com/OSGeo/libgeotiff"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        if not self.options.shared or self.options["proj"].shared: # because PROJ is a C++ library with C API
+            del self.settings.compiler.libcxx
```

Comment:
Why is this deleted conditionally to the shared option?

Reply:
because libgeotiff is a C library, while proj is a C++ library. But anyway I need to remove `self.options["proj"].shared` because it's unknown in `configure`

Reply:
I will be able to unconditionally remove libcxx in libgeotiff recipe with these PR https://github.com/conan-io/conan-center-index/pull/1529 and https://github.com/conan-io/conan-center-index/pull/1530.

Reply:
@danimtb 

#1529 has been merged, therefore `self.settings.compiler.libcxx` is now removed unconditionally.